simul/simul.go

```diff
@@ -1,9 +1,26 @@
 /*
-Package simul enables protocols and services to be run on different platforms,
-like:
+Package simul allows for easy simulation on different platforms. THe following platforms
+are available:
 
-    - localhost - for up to 100 nodes
-    - mininet - for up to 1000 nodes
-    - deterlab - for up to 50'000 nodes
+	- localhost - for up to 100 nodes
+	- mininet - for up to 1'000 nodes
+	- deterlab - for up to 50'000 nodes
+
+Usually you start small, then work your way up to the full potential of your
+protocol!
 */
 package simul
+
+import "flag"
+
+// Start decides on the '--simul'-flag whether it needs to build or start the
+// simulation.
```

Comment:
I know I'am annoying but can you write down a quick sentence about if / when / how this flag should be given. Like `the flag --simul is usually given by the platform's implementation` or stg.